### ISSUE TITLE
Missing information on the ensemble page of queued/running/failed calculations

### ISSUE THREAD

**Issue by RaphaelRobidas** [cyllab/CalcUS#29]
**Is your improvement related to a problem? Please describe.**
The page of an ensemble created by a failed calculation does not show any useful information:

![image](https://user-images.githubusercontent.com/52597883/219449171-7a76a0c4-bb5f-4440-b457-43469c68f6e0.png)

This is also the case for queued and running calculations.

**To Reproduce**
1. Launch a calculation
2. Immediately kill the calculation and wait for it to be stopped
3. Click on the killed calculation order to go to the ensemble page

**Describe the solution or change you'd like**
The ensemble page should display some useful information, such as:
- The parameters of the launched or failed calculation
- Instead of the empty "Geometries" section, a message that specifies that there is no data to show currently

**Comment by Omkar0803** [comment]
From where can I get the failed calculations info to populate on `/ensemble/{{failed_id}}`

**Comment by RaphaelRobidas** [comment]
@Omkar0803 You can launch a conformational search on a random large molecule (so that it would take a long time to complete), then go under the calculation order details and kill the calculation. That will create an ensemble such as shown in my post

**Comment by Omkar0803** [comment]
From where can I get the parameters of failed calculations in order to populate on the above page?

**Comment by RaphaelRobidas** [comment]
The parameters will be in the `parameters` field of the corresponding `CalculationOrder` which generated the ensemble. You can look in the templates, the parameters are already parsed properly for successful calculations.

**Comment by Omkar0803** [comment]
This parameters you are saying?
![Screenshot from 2023-04-18 02-53-35](https://user-images.githubusercontent.com/97947439/232613937-65bb8af1-d7ab-46e4-8fe0-fd65a06ade89.png)

**Comment by Omkar0803** [comment]
I just removed the unnecessary things for failed calculations, just need to add the parameters!
![Screenshot from 2023-04-18 03-20-51](https://user-images.githubusercontent.com/97947439/232618786-d28e3ce5-e1b0-43de-9f27-064ca2cdff60.png)

**Comment by RaphaelRobidas** [comment]
@Omkar0803 Yes, those are the parameters that are missing for failed calculations. Looks good so far!

**Comment by Omkar0803** [comment]
@RaphaelRobidas  I have been in the midst of my exams which ends on 4th of May, this have taken up much of my time and energy. Unfortunately, this has left me with limited capacity to work on the project. I apologize for any inconvenience or disruption this may have caused to the project's timeline.
I want to assure you that my commitment to the project remains strong, and I plan to resume my contributions as soon as my exams are completed.
